inc/classes/class-imagify-filesystem.php

```diff
@@ -713,7 +722,109 @@ public function is_root( $path ) {
 	}
 
 	/**
-	 * Get a clean value of ABSPATH that can be used in string replacements.
+	 * Get the path to the site's root.
+	 *
+	 * @since  1.8.1
+	 * @access public
+	 * @see    get_home_path()
+	 * @author Grégory Viguier
+	 *
+	 * @return string
+	 */
+	public function get_site_root() {
```

Comment:
you didn't have any issue with different setups and this function?

When I tried to copy get_home_path() in WP Rocket, it didn't return the values I expected

Reply:
So far it worked as expected with everything I tested.

Reply:
The only problem with `get_home_path()` is on multisite (surprise!), especially when not on the main site. That's why I use this modified version of it, instead of the real one.